cockpit/angular.json

```diff
@@ -0,0 +1,196 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "newProjectRoot": "projects",
+  "projects": {
+    "varkes-ui-app": {
+      "root": "",
+      "sourceRoot": "src",
+      "projectType": "application",
+      "prefix": "assets",
+      "schematics": {},
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:browser",
+          "options": {
+            "outputPath": "dist/varkes-ui-app",
+            "index": "src/angular.html",
+            "main": "src/main.ts",
+            "polyfills": "src/polyfills.ts",
+            "tsConfig": "src/tsconfig.app.json",
+            "assets": [
+              "src/favicon.ico",
+              "src/assets",
+              "src/app",
+              "src/index.html",
+              "src/logout.html",
+              {
+                "glob": "fiori-fundamentals.min.css",
+                "input": "node_modules/fiori-fundamentals/dist",
+                "output": "/fiori-fundamentals"
+              },
+              {
+                "glob": "fonts/**",
+                "input": "node_modules/fiori-fundamentals/dist",
+                "output": "/fiori-fundamentals"
+              },
+              {
+                "glob": "SAP-icons.*",
+                "input": "node_modules/fiori-fundamentals/dist",
+                "output": "/fiori-fundamentals"
+              },
+              {
+                "glob": "**",
+                "input": "node_modules/@kyma-project/luigi-core",
+                "output": "/luigi-core"
+              },
+              {
+                "glob": "luigi-client.js",
+                "input": "node_modules/@kyma-project/luigi-client",
+                "output": "/luigi-client"
+              }
+            ],
+            "styles": [
+              "src/styles.css",
+              "node_modules/fiori-fundamentals/dist/fiori-fundamentals.min.css"
+            ],
+            "scripts": [],
+            "es5BrowserSupport": true
+          },
+          "configurations": {
+            "node": {
+              "fs": "empty"
+            },
+            "production": {
+              "fileReplacements": [
+                {
+                  "replace": "src/environments/environment.ts",
```

Comment:
Are we using the environment mechanism at all? If not, please remove that lines and the related files